server/models/services/TestPlanRunService.js

```diff
@@ -114,6 +114,20 @@ const createTestPlanRun = async (
     testResultAttributes = TEST_RESULT_ATTRIBUTES,
     userAttributes = USER_ATTRIBUTES
 ) => {
+    // shouldn't have duplicate entries for a tester
+    const existingTestPlanRuns = await getTestPlanRuns(
+        '',
+        {
+            tester,
+            testPlanReport
+        },
+        testPlanRunAttributes,
+        testPlanReportAttributes,
+        testResultAttributes,
+        userAttributes
+    );
+    if (existingTestPlanRuns.length) return existingTestPlanRuns[0];
```

Comment:
We'll need to implement this rule (no duplicate test plan runs) at the GraphQL level, so probably just throwing an error is better at this level. I'm a fan of throwing errors 😈

Reply:
Yep, will definitely lock that into a future migration!